src/k8s/pkg/utils/k8s/status.go

```diff
@@ -40,5 +36,5 @@ func (c *Client) ClusterReady(ctx context.Context) (bool, error) {
 		}
 	}
 
-	return false, nil
+	return false, nil // Cluster has no ready nodes
```

Comment:
Avoid inline comments like this, which would mean that someone has to lookup the implementation. Instead make sure they are in the go doc.